Makefile.js

```diff
@@ -882,6 +882,11 @@ target.checkGitCommit = function() {
 
     // Only check non-release messages
     if (!semver.valid(commitMsgs[0])) {
+        if (commitMsgs[0].slice(0, commitMsgs[0].indexOf("\n")).length > 72) {
```

Comment:
You are correct. I just verified that it doesn't add line break, it truncates, so I think this should work.